### ISSUE TITLE
Broken in Safari

### ISSUE THREAD

**Issue by timpulver** [focus-trap/focus-trap#783]
focus-trap currently does not work in Safari. In my current project the focus is trapped fine in Chrome, but in Safari not all elements can be cycled using the `TAB` key.

The same can be seen in the demo (https://focus-trap.github.io/focus-trap/). When clicking the *activate trap* button in the *default behavior* section, the `TAB` key cannot be used to cycle between the elements in Safari (the focus is stuck).

Safari version: Version 15.6.1

**Comment by stefcameron** [comment]
@timpulver Thank you for reporting this. I see the behavior, will investigate.

**Comment by tpu-ableton** [comment]
@stefcameron thanks for looking into it!

**Comment by timpulver** [comment]
@stefcameron thanks for looking into it!

**Comment by stefcameron** [comment]
@timpulver I'm shaking my head at Apple/Safari on this, but it is what it is.

So I'm tested this on Safari `Version 16.0 (17614.1.25.9.10, 17614)` and I saw what you saw.

But then I tried just tabbing through the page normally, without the trap enabled, expecting to go through the __normal__ tab order that all other browsers have, based on common heuristics about the DOM structure, and the fact that links are tabbable by default -- and to be sure, `tabbable` does identify those links as tabbable and focusabled in Safari (as it should).

To my amazement, I went from "activate trap" straight past the links to "deactivate trap"!!

I'll admit I don't spend much time in Safari (I should more for this library...) so my settings are the defaults, which is where Safari __does not__ tab through all elements on the page. 🤦‍♂️ I guess to save you time? No idea who thought this was a good default behavior...

But thankfully, there's a setting to fix this, and enabling that option makes things work again:

![focus-trap-bug](https://user-images.githubusercontent.com/2855350/190527518-04b9828b-706f-481a-b511-e4cfda0f3ac6.gif)

So enabling that should fix your issue. focus-trap is working just fine...

I will fix this issue by adding a mention about this setting in the README.
